### DISCUSSION TITLE
rAudio 1 released.

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#1] [General]
**rAudio 1**

Released.

Please use this **Discussions** for comments and bug reports.

rern

**Comment by Roland-B** [comment]
Just tested v quickly..

Remove volume control from GUI doesn't work - When disabling doesnt actually remove the volume control from GUI (I'm Using USB DAC).

Spotify cannot be enabled.. Toggles on, but if you move away from settings page and back it shows as off.

When testing webradio (I have only tried webradio, nothing else) - the GUI somehow interferes with playback. Get cut-outs, and is worse when you have GUI on screen or go between screens causes it to cut out also.

RolandB (Raspberry Pi 4 Model B Rev 1.2)

**Reply by rern** [reply]
Disable volume from Playback or MPD page?

Spotify - Fix with today's update.

Cut-out GUI - Post a screenshot.

**Reply by Roland-B** [reply]
Disable volume from Playback or MPD page?

On previous versions when MPD > Bit Perfect - No volume is selected, R6 used to remove the volume control from the GUI automatically. I have now removed by choosing Playback > Untick Volume, but have not needed to do this before, I'm not sure if this is intended or not.

Volumio is now working. Thanks.

Cut-out GUI - Post a screenshot.

Cannot post screenshot for an audio cut-out issue?  Anyhow, I am trying now after update and it seems ok at the moment. 

Thanks as always Rern.

RolandB

**Comment by klassikmann** [comment]
Using the GUI on my HTC U11, resolution is 1440x2560, there is a slight overlap between cover and volume knob.
![screenshot](https://user-images.githubusercontent.com/25383513/102688402-c5ba1900-41f6-11eb-8874-8f0a0e469afb.jpg)

EDIT 20201221: Solved with today's update

**Reply by cYCL157** [reply]
I have this issues on my iphone 12 pro as well.

**Reply by rern** [reply]
Fix with today's update.

**Comment by vip5912** [comment]
I have Cirrus Logic Audio Card and Raspberry Pi 2:
Hardware Raspberry Pi 2 Model B Rev 1.1
Output Device Generic Cirrus Logic WM5102
After boot I go to System, select "Generic Cirrus Logic WM5102", reboot -> no sound.
I can't select output (Headset or Line-out).
I changed mixer on MPD page to "HPOUT1 Digital" -> no GUI.
I remarked /etc/mpd.conf:
```
audio_output {
	name           "RPi-Cirrus"
	device         "hw:0,0"
	type           "alsa"
	auto_resample  "no"
	auto_format    "no"
	mixer_type     "" <===!!!!
}
```
systemctl status mpd:
```
Dec 19 16:33:28 rAudio systemd[1]: Starting Music Player Daemon...
Dec 19 16:33:28 rAudio taskset[1882]: exception: Decoder plugin 'wildmidi' is unavailable: configuration file does not exist: /etc/timidity/timidity.cfg
Dec 19 16:33:28 rAudio taskset[1882]: exception: Failed to configure output in line 23; Unrecognized mixer type
Dec 19 16:33:28 rAudio systemd[1]: mpd.service: Main process exited, code=exited, status=1/FAILURE
Dec 19 16:33:28 rAudio systemd[1]: mpd.service: Failed with result 'exit-code'.
Dec 19 16:33:28 rAudio systemd[1]: Failed to start Music Player Daemon.
```

**Reply by rern** [reply]
Please get today's update and try again.

**Comment by EarsofTin** [comment]
Downloaded,imaged and installed this morning then quick tested the features I routinely use.  Only difference noted so far is (as others have mentioned) the "/" character is not accepted in defining a NAS path.  The Bluetooth as sender issue remains from Re+6.

As for naming this brilliant piece of work "rAudio" sounds a bit generic a could be difficult to find with online searches.  If naming rights are still available may I suggest "PrunePi"

rAudio 1 • 20201219
Raspberry Pi 3 Model B Plus Rev 1.3
BCM2837B0 • 4 Cortex-A53 @ 1.4GHz • 1GB
HiFiBerry DAC+
5.4.83-1-ARCH
/mnt/MPD/NAS/Raspberry NAS • 698GB/932GB

**Comment by AndyFx12** [comment]
Bluetooth as sender works with my JBL Flip-4 speaker. But I had to disconnect/reconnect after pairing.
First second of fresh started playback is missing - no gap on track-change, though.
Stopping BT-playback to speaker switches coverart to rAudio-default.

regards, Andy

**Reply by rern** [reply]
Fix with today's update.

**Reply by AndyFx12** [reply]
Hi rern,

thanks for your update. Most is fixed - one issue is still there: Stopping playback with BT-speaker connected -> Generic rAudio-Coverart is shown.

**Comment by R101G** [comment]
@rern 

On the playback page the elapsed time/track time and progress bar do not always reset when moving to the next track. If I click the "next track" icon, it corrects the display for the next track, but subsequent tracks have no time information or progress bar displayed.

This is with just the artwork, progress bar and top/bottom bars shown on the playback page.

**Reply by rern** [reply]
Fix with today's update.

**Comment by EarsofTin** [comment]
Problem with "/" character in mounted drives paths. Not a new problem but it has become more consequential to me.  My workaround was to manually mounted drives via command line but I learned manual mounts don't survive a reboot.  So every reboot (and presumably power interruption means remount and then a library rebuild (2 hours).  Maybe I have too much music.

**Reply by cYCL157** [reply]
i manually edited /etc/fstab so that it survived reboots -- like this:

//1.2.3.4/music/Albums /mnt/MPD/NAS/Albums cifs username=user,password=pass,uid=45,gid=45,iocharset=utf8

**Reply by EarsofTin** [reply]
Groovy.  Thanks for the tip.

**Reply by rern** [reply]
Fix with today's update.

**Comment by cYCL157** [comment]
i noticed today after applying the 20201221 update that when i reboot using the menu option, it does reboot but the ui never seems to detect the pi back up until i click on the ui.  I did this twice just to validate.  it used to refresh automatically before.

**Comment by cYCL157** [comment]
i keep noticing that the ui stops updating after awhile when having the page running in chrome on my windows desktop.  I keep it open all of the time and most of the time when I switch to it, it's current, but sometimes it has stopped updating.  I haven't noticed any errors in the dev tools console.

**Reply by rern** [reply]
The UI stops most activities while not currently focused / visible.
